src/scss/style.scss

```diff
@@ -275,6 +275,11 @@ button + button {
   margin-left: 4px;
 }
 
+.label-button {
+  pointer-events: none;
```

Comment:
are you sure you want to disable point-events? doesn't that disable the indicator it's clickable?